blocks/init/src/blocks/custom/column/components/column-responsive-tab-content.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-unused-vars */
+
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { SelectControl, RangeControl, ToggleControl, Icon } from '@wordpress/components';
+import globalSettings from './../../../manifest.json';
+
+export const ColumnResponsiveTabContent = (props) => {
+  const {
+    width,
+    onChangeWidth,
+    hide,
+    onChangeHide,
+    offset,
+    onChangeOffset,
+    verticalAlign,
+    onChangeVerticalAlign,
+  } = props;
+
+  const widthOptions = {
+    min: -1,
+    max: globalSettings.globalVariables.maxCols,
+    step: 1,
+    initial: globalSettings.globalVariables.maxCols,
+  };
+
+
+  return (
+    <Fragment>
+      {onChangeWidth && (
+        <RangeControl
+          label={
+            <Fragment>
+              <Icon icon={() => <svg height="20" viewBox="0 0 20 20" width="20" xmlns="http://www.w3.org/2000/svg"><path d="m18.4375 4v5.61390625l-3.9939844-3.99398437-1.1048828 1.10484374 2.1190234 2.11898438h-10.9304687l2.11902344-2.11898438-1.10492188-1.10484374-3.97878906 3.97878906v-5.59871094h-1.5625v11.328125h1.5625v-5.67683594l3.97878906 3.97878904 1.10492188-1.1048437-2.11902344-2.1189844h10.9304687l-2.1190234 2.1189844 1.1048828 1.1048437 3.9939844-3.99398435v5.69203125h1.5625v-11.328125z" /></svg>} />
```

Comment:
These inline icons are kinda ugly, could you move the markup to `assets/icon.svg` (or `.html`) and then import here? I assume it won't work without some kind of webpack svg-loader but it might not be a bad idea to add one. 🤷‍♂️